libtransmission/crypto-utils.cc

```diff
@@ -114,7 +114,11 @@ int tr_rand_int_weak(int upper_bound)
         init = true;
```

Comment:
@mikedld arc4random has no seed initialization equevalent, should I also wrap code above in `#else` branch?